openmetadata-ui/src/main/resources/ui/src/components/Users/UserProfileIcon/UserProfileIcon.component.tsx

```diff
@@ -317,7 +327,8 @@ export const UserProfileIcon = () => {
           {isImgUrlValid ? (
             <img
               alt="user"
-              className="app-bar-user-avatar"
+              className="app-bar-user-profile-pic"
```

Comment:
As we are changing the class name here so can you update the class name in the styling file as well?